pkg/repo/repotest/server.go

```diff
@@ -35,22 +35,21 @@ import (
 //
```

Comment:
Good point. I figure this package may be useful on its own for others who want to mock a chart repository, though. Most things in `internal` are all packages we definitely don't want SDK consumers to use (like `internal/version` or `internal/test`, which both rely heavily on Helm's internal architecture to work properly). It could be argued that `repotest` could be useful in certain scenarios like the charts-testing project.

Thoughts?

Reply:
I did not consider the external testing needs for this one, so let's keep it here